README.md

```diff
@@ -41,7 +41,7 @@ Full documentation [here](https://danielcaldas.github.io/react-d3-graph/docs/ind
 [![https://nodei.co/npm/YOUR-MODULE-NAME.png?downloads=true&downloadRank=true&stars=true](https://nodei.co/npm/react-d3-graph.png?downloads=true&downloadRank=true&stars=true)](https://www.npmjs.com/package/react-d3-graph)
 
 ```bash
-npm install d3@^5.5.0      # if you don't have d3 already
+npm install d3@^6.2.0      # if you don't have d3 already
```

Comment:
Please drop the changes here as this should be pushed within the release commit for easy tracking (e.g. [commit for release 2.5.0](https://github.com/danielcaldas/react-d3-graph/commit/b476ef5bfff01a15813a201d2f08d450d5e288ae))